src/atoms/Elbil24ProButton.js

```diff
@@ -0,0 +1,54 @@
+import React  from 'react';
+import styled from '@emotion/styled';
+
+import { getColor } from '../utils';
+
+const Svg = styled.svg`
+  box-sizing: border-box;
+
+	& .st0 {
+		fill: ${getColor('white')}
+	}
+
+	& .st1 {
+		fill: ${getColor('secondary')}
+	}
+`;
+
+/* eslint-disable max-len */
+const Elbil24ProButton = props => (
+	<Svg
```

Comment:
Remove ids (from the paths as well), add <title>

Reply:
Ups.  Forgot to remove those :(